src/Components/animation/DisplayedElement.js

```diff
@@ -161,6 +161,27 @@ class DisplayedElement extends React.Component {
 }
 
 const _initializeStateFromProps = props => {
+  /* Get the element width */
+  var width = 0;
+
+  const playerRadius = Math.min(props.animationWidth, props.animationHeight) / 12;
+  const discRadius = playerRadius / 1.5;
+  const coneSize = playerRadius / 2;
+
+  switch (props.type) {
+    case 'defense':
+    case 'offense':
+      width = playerRadius;
+      break;
+    case 'disc':
+      width = discRadius;
+      break;
+    case 'triangle':
+      width = coneSize / 2;
+  }
+
+  width /= 2;
```

Comment:
Then why not doing `/ 24` at the beginning ?

Reply:
Yes would be better